### PR TITLE
change revoke to deactivate

### PR DESCRIPTION
**Pull request by isaacJChen**

### CODE REVIEW

**Comment by isaacJChen** [comment]
#321

**Comment by OR13** [comment]
Looks good to me

File: lib/core/versions/latest/AnchorFile.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import JsonAsync from './util/JsonAsync';
 import Multihash from './Multihash';
 import ProtocolParameters from './ProtocolParameters';
 import RecoverOperation from './RecoverOperation';
-import RevokeOperation from './RevokeOperation';
+import DeactivateOperation from './DeactivateOperation';
```
Comment (thehenrytsai) [inline lib/core/versions/latest/AnchorFile.ts:11]: Nit: sort.... LOL.

File: lib/core/Resolver.ts [hunk 2]
```diff
@@ -25,21 +25,21 @@ export default class Resolver {
     console.info(`Resolving DID unique suffix '${didUniqueSuffix}'...`);
```
Comment (thehenrytsai) [inline lib/core/Resolver.ts:25]: Can we quickly update the revoke references in protocol.md? If you want, you can remove the descriptive sections, but keep the API sections as the new spec will not specify the API.

File: lib/core/versions/latest/BatchWriter.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ import Operation from './Operation';
 import OperationType from '../../enums/OperationType';
 import ProtocolParameters from './ProtocolParameters';
 import RecoverOperation from './RecoverOperation';
-import RevokeOperation from './RevokeOperation';
+import DeactivateOperation from './DeactivateOperation';
```
Comment (thehenrytsai) [inline lib/core/versions/latest/BatchWriter.ts:16]: Nit: sort for all occurrences... painful I know... but that's the convention.... unless we just say forget about sorting altogether...

**Pull request merged**